real_estate/models/estate_property.py

```diff
@@ -0,0 +1,39 @@
+from odoo import fields, models
+import datetime
+
+class RealEstateProperty(models.Model):
+
+    _name = "estate_property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(string='Property Name', required=True, help="Enter the name of the property")
```

Comment:
```suggestion
    name = fields.Char(
        string="Property Name",
        required=True,
        help="Enter the name of the property"
    )
```
Apply the string applications comment as I mentioned above wherever applicable.
Did you know what string parameter does here?
Also I see some readability issue here, try to follow [PEP8](https://peps.python.org/pep-0008/#maximum-line-length) guidelines 😄